test/index.ts

```diff
@@ -96,6 +96,34 @@ export async function run_tests(tests: Test[]) {
     }
 }
 
+export async function deploy_and_generate(
```

Comment:
I think we should consider getting rid of this, it only works for single canisters, thus many examples will still need a different solution. This happened in the past, and it became simpler to just copy-paste across all examples